spanner/tests/system/test_system.py

```diff
@@ -728,6 +728,33 @@ def _unit_of_work(transaction, test):
 
         return session, committed
 
+    def test_read_with_multiple_keys_index(self):
+        ROW_COUNT = 10
+        EXTRA_DDL = ['CREATE INDEX name ON contacts(first_name, last_name)']
+        MY_COLUMNS = self.COLUMNS[1], self.COLUMNS[2]
+        pool = BurstyPool()
+
+        temp_db = Config.INSTANCE.database(
+            'test_read_multi_keys_index', ddl_statements=DDL_STATEMENTS + EXTRA_DDL,
```

Comment:
Please make sure your lines are under 80 chars. I'd also suggest configuring your editor so this is "obvious".